AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/sample.json

```diff
@@ -10,11 +10,17 @@
   "ciTests": {
   	"linux": [
     {
-  		"env": ["source /opt/intel/oneapi/setvars.sh --force", "conda create -n aikit-modin-test -c intel -c conda-forge matplotlib runipy intel-aikit-modin", "source activate aikit-modin-test"],
+  		"env": [],
```

Comment:
For some reason, placing environments setup and the notebook execution in different cells (`env` and `steps`) caused the environment wasn't applied (this was indicated by the "Import error: couldn't find 'modin'" error at CI logs).

Placing environment setup steps and the actual notebook execution in a single cell resolved the problem.